interceptor/opencensus/opencensus.go

```diff
@@ -88,20 +90,28 @@ func (oci *OCInterceptor) Export(tes agenttracepb.TraceService_ExportServer) err
 		return errTraceExportProtocolViolation
 	}
 
-	var lastNonNilNode *commonpb.Node
+	spansMetricsFn := internal.NewReceivedSpansRecorder(context.Background(), "opencensus")
```

Comment:
Use tags from the current context (they include tags emitted by the oc-library-agent-exporter)?

Reply:
Great reminder, that'd entail changing that to `tes.Context()`, thanks.